### PR TITLE
remove file locking

### PR DESCRIPTION
**Pull request by deads2k**
@sttts I think this is stopping me from getting my cluster up

### CODE REVIEW

**Comment by deads2k** [comment]
```
[root@test1-bootstrap kubernetes]# crictl logs cd3f65d3a97a6
flock: cannot open lock file /var/lock/api-server.lock: Permission denied

```

**Comment by sttts** [comment]
Please also remove the init container.

File: bindata/bootkube/bootstrap-manifests/kube-apiserver-pod.yaml [hunk 1]
```diff
@@ -13,37 +13,18 @@ spec:
   hostNetwork: true
   securityContext:
     supplementalGroups: [65534]
```
Comment (sttts) [inline bindata/bootkube/bootstrap-manifests/kube-apiserver-pod.yaml]: this group can go as well

File: bindata/bootkube/bootstrap-manifests/kube-apiserver-pod.yaml [hunk 2]
```diff
@@ -13,37 +13,18 @@ spec:
...
   containers:
   - name: kube-apiserver
     image: {{ .Image }}
     imagePullPolicy: {{ .ImagePullPolicy }}
-    command: ["/usr/bin/flock", "--exclusive", "--timeout=60", "/var/lock/api-server.lock", "-c"]
+    command: ["hypershift", "openshift-kube-apiserver"]
     args:
-    - exec hypershift openshift-kube-apiserver --config=/etc/kubernetes/config/{{ .ConfigFileName }}
+    - --config=/etc/kubernetes/config/{{ .ConfigFileName }}
     securityContext:
```
Comment (sttts) [inline bindata/bootkube/bootstrap-manifests/kube-apiserver-pod.yaml]: this whole block can go

**Pull request merged**